build_common.ps1

```diff
@@ -271,10 +262,6 @@ class BuildProject {
 			# Loop through all files in subdirectories and fail the build if any filenames are missing inside the project file
 			Get-ChildItem $this.modSrcRoot -Directory | Get-ChildItem -File -Recurse |
 			ForEach-Object {
-				# Compiletest file is allowed to be missing because it's not commited and manually edited
-				if ($_.Name -Match "_Compiletest\.uc$") {
-					return
-				}
```

Comment:
Looking at the pre-make callback in CHL, how does this check pass with the new implementation?

Reply:
I see. I think it's a good idea to add a comment about it in CHL's `build.ps1` as it's very easy to get confused about different but similar looking paths that are used throughout the build